targets/targets.json

```diff
@@ -334,9 +334,26 @@
             "SLEEP",
             "SPI",
             "SPISLAVE",
-            "STDIO_MESSAGES"
+            "STDIO_MESSAGES",
+            "USTICKER"
         ],
-        "default_lib": "small",
+        "overrides": {
+            "tickless-from-us-ticker": true,
+            "boot-stack-size": "0x300"
+        },
+        "supported_c_libs": {
+            "arm": [
+                "std",
```

Comment:
Is there enough memory to support the "std" libraries?
If this board does not run in RTOS mode then we could only list the small libraries in `supported_c_libs`.

Reply:
Agreed. I removed `std` lib from `arm` and `gcc_arm`.